# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: docshell/base/CanonicalBrowsingContext.cpp
# Commit: 1177913e1edf
# Full Hash: 1177913e1edf443da73d438f95a334e466b623e2
# Author: Manuel Bucher <manuel@mozilla.com>
# Date: 2022-12-02 21:26:33
# Regressor Bug: 1771867
# File Overlap Count: 1
# Description:
#   Bug 1771867 - Early Hints Phase 2 - Part 8: Add list of open channels from OngoingEarlyHints to RedirectToRealChannelArgs r=necko-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D161179
# ==============================================================================

diff -r d33ccbbf407d -r 1177913e1edf docshell/base/CanonicalBrowsingContext.cpp
--- a/docshell/base/CanonicalBrowsingContext.cpp	Fri Dec 02 09:45:26 2022 +0000
+++ b/docshell/base/CanonicalBrowsingContext.cpp	Fri Dec 02 09:45:27 2022 +0000
@@ -1297,10 +1297,6 @@
   mFullyDiscardedListeners.AppendElement(std::move(aListener));
 }
 
-net::EarlyHintsService* CanonicalBrowsingContext::GetEarlyHintsService() {
-  return &mEarlyHintsService;
-}
-
 void CanonicalBrowsingContext::AdjustPrivateBrowsingCount(
     bool aPrivateBrowsing) {
   if (IsDiscarded() || !EverAttached() || IsChrome()) {